src/plugins/intel_gpu/src/plugin/ops/non_max_suppression.cpp

```diff
@@ -189,6 +192,90 @@ static void CreateNonMaxSuppressionIEInternalOp(ProgramBuilder& p, const std::sh
     }
 }
 
-REGISTER_FACTORY_IMPL(internal, NonMaxSuppressionIEInternal);
+static void CreateNonMaxSuppressionOp(ProgramBuilder& p, const std::shared_ptr<ov::op::v9::NonMaxSuppression>& op) {
```

Comment:
random spot) could you add test case for dynamic-shaped nms?

Reply:
@isanghao dynamic shape NMS test is already implemented under NmsLayerGPUTest class.